io.ente.auth.yml

```diff
@@ -0,0 +1,234 @@
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/WebPlatformForEmbedded/libwpe/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag
+              url-query: '"https://github.com/WebPlatformForEmbedded/libwpe/releases/download/\($tag)/libwpe-\($version).tar.xz"'
+      - name: wpebackend-fdo
+        buildsystem: meson
+        builddir: true
+        config-opts:
+          - --buildtype=plain
+        sources:
+          - type: archive
+            url: https://github.com/Igalia/WPEBackend-fdo/releases/download/1.14.2/wpebackend-fdo-1.14.2.tar.xz
+            sha256: 93c9766ae9864eeaeaee2b0a74f22cbca08df42c1a1bdb55b086f2528e380d38
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/Igalia/WPEBackend-fdo/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag
+              url-query: '"https://github.com/Igalia/WPEBackend-fdo/releases/download/\($tag)/wpebackend-fdo-\($version).tar.xz"'
+
+  - name: libsodium
+    sources:
+      - type: archive
+        url: https://github.com/jedisct1/libsodium/archive/1.0.18-RELEASE.tar.gz
+        sha256: b7292dd1da67a049c8e78415cd498ec138d194cfdb302e716b08d26b80fecc10
+        x-checker-data:
+          type: anitya
+          project-id: 1728
+          url-template: https://github.com/jedisct1/libsodium/archive/$version.tar.gz
+
+  - name: ente_auth
+    buildsystem: simple
+    build-commands:
+      - bsdtar --to-stdout -xf ente-auth.deb data.* | bsdtar -xf -
+      - rm -rf usr/share/icons/hicolor/
+      - mv usr/share/* ${FLATPAK_DEST}/share/
+      - desktop-file-edit --remove-key="Version" ${FLATPAK_DEST}/share/applications/ente_auth.desktop
+      - install -dm755 ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - cp 128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/ente_auth.png
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libsqlite3.so.0 ${FLATPAK_DEST}/lib/libsqlite3.so
+      - ln -s ${FLATPAK_DEST}/share/ente_auth/ente_auth ${FLATPAK_DEST}/bin/ente_auth
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/ente-io/ente/releases/download/auth-v3.1.3/ente-auth-v3.1.3-x86_64.deb
+        sha256: 03eaabcb14b2af781fb3feec4905b8695422294292546245252567b5500e242c
+        dest-filename: ente-auth.deb
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/ente-io/ente/releases/latest
+          version-query: '[.[].tag_name | select(startswith("auth-v"))] | sort_by(split("-v")[1:])[-1]'
+          url-query: .[].assets[] | select(.name == "ente-auth-" + $version + "-x86_64.deb").browser_download_url
+          is-main-source: true
+      - type: file
+        path: 128x128.png
```

Comment:
There is no icon in the deb ?

Reply:
See https://github.com/flathub/flathub/pull/5605#discussion_r1764076485

(was marked as resolved)